### ISSUE TITLE
wslgit add fails

### ISSUE THREAD

**Issue by bondib** [andy-5/wslgit#62]
C:\Users\bondi\dev\WSIL\healthshield>**wslgit.exe add -A** -- c:\Users\bondi\dev\WSIL\healthshield\tasker\tasker\tasks\move_nodes.py

Results with:

fatal: /mnt/c/Users/bondi/dev/WSIL/healthshield/tasker/tasker/tasks/move_nodes.py: '/mnt/c/Users/bondi/dev/WSIL/healthshield/tasker/tasker/tasks/move_nodes.py' is outside repository

**Comment by andy-5** [comment]
What is the output of `wslgit.exe rev-parse --show-toplevel`? This should print the root directory of the repository (I assume this would be `c:\Users\bondi\dev\WSIL\healthshield`).

**Comment by bondib** [comment]
The output is:

C:\Users\bondi\dev\WSIL\healthshield>C:\Users\bondi\dev\wslgit.exe rev-parse --show-toplevel

**/c/Users/bondi/dev/WSIL/healthshield**

**Comment by andy-5** [comment]
Do you have any custom mount points configured, either via ` wsl.conf` or for example in `.bashrc`?

Your problem looks like a duplicate of #42 or #60.

**Comment by andy-5** [comment]
You could try the new experimental `wslgit-mount-at-root.exe` build from the [latest release](https://github.com/andy-5/wslgit/releases/tag/v0.7.0) which should support your setup.

Can you please report back if this works for you?

**Comment by pjnovas** [comment]
That one worked for me, I had tried `wslgit.exe` (release 0.7.0) adding the environment variable `WSLGIT_MOUNT_ROOT` to `/` and it didn't work, then used `wslgit-mount-at-root.exe` and worked out

My VSCode:
```
Version: 1.36.1 (user setup)
Commit: 2213894ea0415ee8c85c5eea0d0ff81ecc191529
Date: 2019-07-08T22:59:35.033Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
```

Thanks!

**Comment by andy-5** [comment]
Thanks for the feedback, great to hear that the special build from release 0.7.0 works for you.

The `WSLGIT_MOUNT_ROOT` environment variable was only recently added and will be available in the upcoming 0.8.0 release. You can get an alpha build [here](https://ci.appveyor.com/project/andy-5/wslgit/build/artifacts) if you want to try it.
